src/bank.rs

```diff
@@ -897,39 +595,32 @@ impl Bank {
     }
 
     pub fn get_account(&self, pubkey: &Pubkey) -> Option<Account> {
-        self.accounts.load_slow(pubkey)
+        let state = self.live_bank_state();
+        state.load_slow(pubkey)
     }
 
+    pub fn store_slow(&self, pubkey: &Pubkey, account: &Account) {
+        let state = self.live_bank_state();
+        let purge = state.checkpoints.len() == 1;
```

Comment:
Can you help me understand what the significance of `checkpoints.len == 1` and why it determines if we purge an account?

Reply:
added comments